Testing/functional/tests/cypress/integration/ui/timeline/filter.js

```diff
@@ -73,7 +73,7 @@ describe("Filters", () => {
 
         cy.get("[data-testid=filterDropdown]").click();
         cy.get("[data-testid=Immunization-filter]").click({ force: true });
-        cy.get("[data-testid=btnFilterApply]").click();
+        cy.get("[data-testid=btnFilterApply]").click().should("not.exist");
```

Comment:
That line doesn't look quite right. Don't think we need the .click() part?

Reply:
Clicking the apply button on the filter causes the filter to disappear, so this is doing that and then confirming it is gone before continuing.  I'll update the line into separate statements for clarity, e.g.:

```
cy.get("[data-testid=btnFilterApply]").click();
cy.get("[data-testid=btnFilterApply]").should("not.exist");
```

This was needed because our later attempt to reopen the filter wasn't working.  My best guess is the filter popover has some animation to its closing and Cypress tried to reopen it before it had fully closed, which it decided to ignore.